app/main.py

```diff
@@ -2,0 +1,34 @@
+import os
+
+
+def move_file(string: str) -> None:
+    words = string.split()
+    if len(words) != 3:
+        raise TypeError
+    elif words[0] != "mv":
+        raise ValueError
+
+    old_file_name = words[1]
+
+    if words[-1][-1] == "\\" or words[-1][-1] == "/":
+        destination_path = os.path.join(*words[-1])
+        os.makedirs(destination_path, exist_ok=True)
+        return None
+    else:
+        destination_path = os.path.join(*words[-1].split("/"))
```

Comment:
The splitting of `words[-1]` should be done based on the operating system's path separator to ensure cross-platform compatibility. Consider using `os.path.sep` for splitting.